src/Tags/Concerns/QueriesScopes.php

```diff
@@ -8,15 +8,9 @@ trait QueriesScopes
 {
     public function queryScopes($query)
     {
-        $this->parseQueryScopes()
-            ->map(function ($handle) {
-                return app('statamic.scopes')->get($handle);
-            })
-            ->filter()
-            ->each(function ($class) use ($query) {
-                $scope = app($class);
-                $scope->apply($query, $this->params);
-            });
+        $this->parseQueryScopes()->each(function ($handle) use ($query) {
+            $query->appplyScope($handle, $this->params);
```

Comment:
Should this be applyScope? Notice the extra p

Reply:
Yes, thanks. Just spotted that a minute ago.